specification/validated-relationships-service-api.yaml

```diff
@@ -295,10 +301,9 @@ components:
           enum: ["completed"]
         authored: 
           type: string
-          description: The date and/or time that this set of answers were last changed.
-          # On Confluence the example has this as a string but here:
-          # https://simplifier.net/packages/hl7.fhir.r4.core/4.0.1/files/81730/~details
-          # it looks like an object
+          format: date-time
+          description: The date time that this set of answers were last changed.
+          example: "2024-07-15T09:43:03.280Z"
         source: 
           type: object
           description: The person who answered the questions about the subject.
```

Comment:
Can't leave a comment on line 315, but be good to make this align with other descriptions for identifiers please. Suggest:

The NHS number of the person to complete the questionnaire response. This may be a proxy or patient. The primary identifier of a patient, unique within NHS England and Wales. Always 10 digits and must be a [valid NHS number](https://www.datadictionary.nhs.uk/attributes/nhs_number.html).